plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -2084,6 +2093,21 @@ def updateBrushSize(self, value):
             effect = self._scribblesEditorWidget.activeEffect()
             effect.setParameter("BrushAbsoluteDiameter", value)
 
+    def createScribblesROINode(self):
+        scribblesROINode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLMarkupsROINode")
+        scribblesROINode.AddObserver(scribblesROINode.PointPositionDefinedEvent, self.onROIPointPlaced)
+        scribblesROINode.SetName("Scribbles ROI")
+        scribblesROINode.CreateDefaultDisplayNodes()
+        scribblesROINode.GetDisplayNode().SetFillOpacity(0.0)
```

Comment:
Can you update this to have slight opacity to better differentiate inside and outside of the ROI?

e.g.
`scribblesROINode.GetDisplayNode().SetFillOpacity(0.4)` works well on my end.